compiler/transf.nim

```diff
@@ -762,6 +762,10 @@ proc transformCall(c: PTransf, n: PNode): PTransNode =
           inc(j)
       add(result, a.PTransNode)
     if len(result) == 2: result = result[1]
+  elif magic == mAddr:
+    result = newTransNode(nkHiddenAddr, n, 1)
```

Comment:
Huh, how do you know it's ``nkHiddenAddr`` here?

Reply:
it is rather random choice, it can be `nkAddr` if you like. I thought it is good idea to remove `nkAddr` node in future  and leave only `call to mAddr magic` instead.

Reply:
Ah I see. But please make it an ``nkAddr`` then.

Reply:
done